### ISSUE TITLE
I have an issue while i'm installing pytorch3d

### ISSUE THREAD

**Issue by JSWLDG** [Jeff-sjtu/HybrIK#190]
pip install git+ssh://git@github.com/facebookresearch/pytorch3d.git@stable
Collecting git+ssh://****@github.com/facebookresearch/pytorch3d.git@stable
  Cloning ssh://****@github.com/facebookresearch/pytorch3d.git (to revision stable) to /tmp/pip-req-build-_dd862p8
  Running command git clone --filter=blob:none --quiet 'ssh://****@github.com/facebookresearch/pytorch3d.git' /tmp/pip-req-build-_dd862p8
  git@github.com: Permission denied (publickey).
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights
  and the repository exists.
  error: subprocess-exited-with-error
  
  × git clone --filter=blob:none --quiet 'ssh://****@github.com/facebookresearch/pytorch3d.git' /tmp/pip-req-build-_dd862p8 did not run successfully.
  │ exit code: 128
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet 'ssh://****@github.com/facebookresearch/pytorch3d.git' /tmp/pip-req-build-_dd862p8 did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

I want to know what's wrong

**Comment by TooManyStacks** [comment]
Same, here is a complete error log:

import torch
import torch.nn as nn

class CustomModel(nn.Module):
    def __init__(self):
        super(CustomModel, self).__init__()
        
        self.conv1 = nn.Conv2d(in_channels=3, out_channels=64, kernel_size=3, padding=1)
        self.conv2 = nn.Conv2d(in_channels=64, out_channels=128, kernel_size=3, padding=1)
        self.conv3 = nn.Conv2d(in_channels=128, out_channels=256, kernel_size=3, padding=1)
        self.maxpool1 = nn.MaxPool2d(kernel_size=2, stride=2)
        self.dropout1 = nn.Dropout(p=0.5)
        
        self.conv4 = nn.Conv2d(in_channels=256, out_channels=256, kernel_size=3, padding=1)
        self.conv5 = nn.Conv2d(in_channels=256, out_channels=512, kernel_size=3, padding=1)
        self.conv6 = nn.Conv2d(in_channels=512, out_channels=512, kernel_size=3, padding=1)
        self.maxpool2 = nn.MaxPool2d(kernel_size=2, stride=2)
        self.dropout2 = nn.Dropout(p=0.5)
        
        self.conv7 = nn.Conv2d(in_channels=512, out_channels=512, kernel_size=3, padding=1)
        self.conv8 = nn.Conv2d(in_channels=512, out_channels=512, kernel_size=3, padding=1)
        self.maxpool3 = nn.MaxPool2d(kernel_size=2, stride=2)
        self.dropout3 = nn.Dropout(p=0.5)
        
        self.fc = nn.Linear(512, num_classes)  # Adjust 'num_classes' to your task
        
        self.softmax = nn.Softmax(dim=1)
        
    def forward(self, x):
        x = self.dropout1(self.maxpool1(nn.functional.relu(self.conv3(nn.functional.relu(self.conv2(nn.functional.relu(self.conv1(x))))))))
        x = self.dropout2(self.maxpool2(nn.functional.relu(self.conv6(nn.functional.relu(self.conv5(nn.functional.relu(self.conv4(x))))))))
        x = self.dropout3(self.maxpool3(nn.functional.relu(self.conv8(nn.functional.relu(self.conv7(x))))))

        x = x.view(x.size(0), -1)
        x = self.fc(x)
        x = self.softmax(x)
        
        return x

# Create an instance of the model
model = CustomModel()

# Print the model architecture
print(model)

**Comment by TooManyStacks** [comment]
Same error, the mess in the terminal is 27 MB worth of text.

**Comment by biansy000** [comment]
You may refer to https://github.com/Jeff-sjtu/HybrIK/issues/186#issue-1814023855
